akka-stream/src/main/scala/akka/stream/scaladsl/Materialization.scala

```diff
@@ -15,8 +17,25 @@ object Keep {
   private val _both = (l: Any, r: Any) ⇒ (l, r)
   private val _none = (l: Any, r: Any) ⇒ NotUsed
 
+  private val _leftAsync = (l: Any, r: Any) ⇒ Future.successful(l)
```

Comment:
There are useful for the tests, so I would say that's appropriate to have them in the same PR